pom.xml

```diff
@@ -101,7 +101,7 @@
 		<dependency>
 			<groupId>commons-collections</groupId>
 			<artifactId>commons-collections</artifactId>
-			<version>3.1</version>
+			<version>3.0</version>
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `commons-collections` violates your company's policies.

Lifecycle recommends using version [3.2.2](https://search.maven.org/artifact/commons-collections/commons-collections/3.2.2/jar) instead as it does not violate any policies.